lib/zone/function-call.ts

```diff
@@ -100,14 +100,21 @@ function callFunction(
     options: CallOptions): any {
 
     let module: any = null;
-    if (moduleName == null || moduleName.length === 0) {
+    let useAnonymousFunction: boolean = false;
+
+    if (moduleName == null || moduleName.length === 0 || moduleName === 'global') {
         module = global;
-    } else if (moduleName !== '__function') {
+    } else if (moduleName === '__function') {
+        useAnonymousFunction = true;
+    } else {
         module = require(moduleName);
     }
 
     let func = null;
-    if (module != null) {
+    if (!useAnonymousFunction) {
```

Comment:
> (! [](start = 6, length = 3)

Can you switch the order of the if. It is preferred for readability to use positive rather then negative expressions.